### ISSUE TITLE
Suggestions: Success statistics

### ISSUE THREAD

**Issue by ryancabanas** [teuteuf/worldle#291]
Hi.  I'd like to make a suggestion, please.  I think it would be fun to have a statistic showing, for the previous day's country, how many users guessed the country right with a breakdown of how many guesses it took them.  Thanks!

**Comment by NickSto** [comment]
I agree, this would be really cool. But it would require adding an entire server backend, which would like double the size of the codebase and more than double the headache of maintaining it.
